packages/camera/camera_web/lib/src/types/camera_options.dart

```diff
@@ -58,25 +58,28 @@ class CameraOptions {
 class AudioConstraints {
   /// Creates a new instance of [AudioConstraints]
   /// with the given [enabled] constraint.
-  const AudioConstraints({this.enabled = false});
+  const AudioConstraints({this.bitrate, this.enabled = false});
 
   /// Whether the audio track should be enabled.
   final bool enabled;
 
+  /// Audio bitrate
+  final int? bitrate;
+
   /// Converts the current instance to a Map.
-  Object toJson() => enabled;
+  Map<String, dynamic> toJson() =>
+      <String, dynamic>{'enabled': enabled, 'bitrate': bitrate};
```

Comment:
`bitrate` is not a `MediaTrackConstraint`, [according to the docs](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints).

I don't think this value should be passed to the constraints JSON, otherwise it might cause the `getUserMedia` call to be over-constrained and not return any cameras? Please, take a look at these [docs](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) as well!

(There's a cool demo here: https://webrtc.github.io/samples/src/content/peerconnection/constraints)

Reply:
`Because of the way interface definitions in WebIDL work, if a constraint is requested but not supported, no error will occur` [here](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackSupportedConstraints)

but ok, i will extract bitrate from options and pass directly to `Camera`